isis/src/docsys/build/menu.xsl

```diff
@@ -48,7 +48,7 @@
       </li>
     </ul>
     <script><![CDATA[
-      fetch('/versions.json')
```

Comment:
I am still seeing issues with this fetch not working with the Cloudfront. So currently working through this.